### PR TITLE
[12.0][MIG] web_switch_context_warning

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module web_switch_company_warning to v12 from #1288 

cc ~ @Eficent

### CODE REVIEW

**Comment by tbaden** [comment]
could you please fix these travis warnings:
<details><summary>logs</summary>

> ************* Module web_switch_company_warning
> web_switch_company_warning/static/src/js/switch_company_warning.js:7: [W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]
> web_switch_company_warning/static/src/js/switch_company_warning.js:8: [W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]
> web_switch_company_warning/static/src/js/switch_company_warning.js:11: [W7903(javascript-lint), ] Comments should not begin with a lowercase character. [Error/capitalized-comments]
> web_switch_company_warning/static/src/js/switch_company_warning.js:11: [W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]
> web_switch_company_warning/static/src/js/switch_company_warning.js:16: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
> web_switch_company_warning/static/src/js/switch_company_warning.js:19: [W7903(javascript-lint), ] Line 19 exceeds the maximum line length of 80. [Error/max-len]
> web_switch_company_warning/static/src/js/switch_company_warning.js:24: [W7903(javascript-lint), ] Expected space(s) after "if". [Error/keyword-spacing]
> web_switch_company_warning/static/src/js/switch_company_warning.js:33: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
> web_switch_company_warning/static/src/js/switch_company_warning.js:35: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
> web_switch_company_warning/static/src/js/switch_company_warning.js:39: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
> web_switch_company_warning/static/src/js/switch_company_warning.js:43: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:1: [W7903(javascript-lint), ] Use the function form of 'use strict'. [Error/strict]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:2: [W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:3: [W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:5: [W7903(javascript-lint), ] Implicit global variable, assign as global property instead. [Error/no-implicit-globals]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:6: [W7903(javascript-lint), ] Implicit global variable, assign as global property instead. [Error/no-implicit-globals]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:8: [W7903(javascript-lint), ] Use the function form of 'use strict'. [Error/strict]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:8: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
> web_switch_company_warning/static/src/js/switch_company_warning_worker.js:17: [W7903(javascript-lint), ] Line 17 exceeds the maximum line length of 80. [Error/max-len]

</details>

**Comment by pedrobaeza** [comment]
Why this one is called `web_switch_company_warning` and not `web_switch_context_warning`?

**Comment by AdriaGForgeFlow** [comment]
> Why this one is called `web_switch_company_warning` and not `web_switch_context_warning`?

I was waiting for some feedback on the renaming of the module. I assume `web_switch_context_warning` is ok then.

File: web_switch_company_warning/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+    * If the browser don't implement Sharded Worker (http://www.w3.org/TR/workers/#sharedworker), the warning message will not be displayed (there is no polyfill).
```
Comment (Tardo) [inline web_switch_company_warning/readme/ROADMAP.rst]: Sharded -> Shared

**Review by Tardo — APPROVED** [review]
LGTM :+1:

**Review by tarteo — APPROVED** [review]
LGTM

**Pull request merged**